mitodl/ocw-hugo-themes
# allow launching of an externally converted course

#### Pre-Flight checklist

#### What are the relevant tickets?
None

#### What's this PR do?
The `npm run start:course` command currently only supports downloading data from S3.  This PR allows you to optionally bypass this for working with data locally converted in `ocw-to-hugo` manually.  This is useful for development if you are working on testing a PR with a specific branch of `ocw-to-hugo`.

#### How should this be manually tested?
 - Clone `ocw-to-hugo`, read the readme if you've never used it before and convert any number of courses
 - See the updated readme and example env, set your variables to point at your local copy of `ocw-to-hugo` and set `DOWNLOAD` to `0`
 - Set `OCW_TEST_COURSE` to one of the courses you downloaded in `ocw-to-hugo`
 - Run `npm run start:course`
 - Verify that you don't see any log entries saying that a course is being downloaded, and ensure that the course site is available at http://localhost:3000
